src/main/java/it/gov/pagopa/gpd/payments/pull/models/PaymentNotice.java

```diff
@@ -53,7 +57,7 @@ public class PaymentNotice implements Serializable {
             required = true)
     private PaymentNoticeStatus status;
     @Schema(description = "Date of update of the Debt Position", required = true)
-    private LocalDate lastUpdateDate;
+    private LocalDateTime lastUpdateDate;
```

Comment:
Why was there a need for this update?

Reply:
Because there was a mismatch between the data type on the db and the entity. Hibernate was throwing the following error: ERROR [io.qu.hi.or.ru.sc.SchemaManagementIntegrator] (Hibernate post-boot validation thread for <default>) Failed to validate Schema: Schema-validation: wrong column type encountered in column [last_updated_date] in table [apd.payment_position]; found [timestamp (Types#TIMESTAMP)], but expecting [date (Types#DATE)]